examples/hosts

```diff
@@ -0,0 +1,16 @@
+# To connect using a specific user (and elevate to root with sudo later),
+# add `ansible_ssh_user=<your user>` next to <your domain> below.
+#
+# For improved Ansible performance, SSH pipelining is enabled by default in `ansible.cfg`.
+# If this causes SSH connection troubles, disable it by adding `ansible_ssh_pipelining=False`
+# to the host line below or by adding `ansible_ssh_pipelining: False` to your variables file.
+#
```

Comment:
pipelining is not enabled by default, when I understand it correctly:
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/ssh_connection.html#parameter-pipelining
https://docs.ansible.com/ansible/latest/reference_appendices/config.html#ansible-pipelining

Also, I would suggest to use `pipelining` variable in the `ssh_connection` section of ansible.cfg.

Reply:
You are exactly right, I meant to change that to say "For improved Ansible performance, enable SSH pipelining by adding  `ansible_ssh_pipelining=True` in your `ansible.cfg`", but I guess I forgot. Nice catch!